staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -389,18 +390,24 @@ func interfaceFromUnstructured(sv, dv reflect.Value) error {
 }
 
 func (c *converterImpl) ToUnstructured(obj interface{}) (map[string]interface{}, error) {
```

Comment:
this func really need a godoc.

Reply:
are there tests for it? Just to be sure that the non-trivial change below does not have subtle bugs.

Reply:
This method is exercised in tests at least here:

https://github.com/kubernetes/kubernetes/blob/master/pkg/api/unstructured_test.go
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter_test.go

Probably also tested indirectly.

Reply:
Also I'm adding one more test in #51505.

Reply:
There is godoc on the interface https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go#L41 and I've just added godoc to the impl methods.